src/containerapp-preview/azext_containerapp_preview/tests/latest/test_containerapp_preview_scenario.py

```diff
@@ -24,16 +24,6 @@ def setUp(self):
         cmd = ['azdev', 'extension', 'add', 'k8s-extension']
         run(cmd, check=True)
 
-    def tearDown(self):
-        cmd = ['azdev', 'extension', 'remove', 'containerapp']
```

Comment:
for tests case be more stable and faster

Reply:
@wangzelin007 

This CI test failed [Pipelines - Run 20230712.36 logs (azure.com)](https://dev.azure.com/azclitools/public/_build/results?buildId=71727&view=logs&jobId=ff866e8d-4875-52ff-9fd3-f181f728432c&j=ff866e8d-4875-52ff-9fd3-f181f728432c&t=64f04c5b-80b8-53a1-8045-e38ac626841b)

only python 3.10 tests failed. And Error as following.

[gw0] [ 50%] FAILED src/containerapp-preview/azext_containerapp_preview/tests/latest/test_containerapp_preview_scenario.py::ContainerappScenarioTest::test_containerapp_preview_e2e

src/containerapp-preview/azext_containerapp_preview/tests/latest/test_containerapp_preview_scenario.py::ContainerappScenarioTest::test_containerapp_preview_environment_type

[gw0] [100%] PASSED src/containerapp-preview/azext_containerapp_preview/tests/latest/test_containerapp_preview_scenario.py::ContainerappScenarioTest::test_containerapp_preview_environment_type

 

ERROR cli.azure.cli.core.azclierror:azlogging.py:212 unrecognized arguments: --environment-type managed

ERROR az_command_data_logger:azlogging.py:213 unrecognized arguments: --environment-type managed

Reply:
And with more and more test cases( with different file), this kind of `azdev extension add containerapp` in a case will be unstable